transactions/btc.ts

```diff
@@ -658,88 +659,88 @@ export async function signOrdinalSendTransaction(
   fee?: BigNumber,
 ): Promise<SignedBtcTx> {
   // Get sender address unspent outputs
-  const btcClient = new BitcoinEsploraApiProvider({
-    network,
-  });
-  const unspentOutputs = await btcClient.getUnspentUtxos(btcAddress);
+  try {
```

Comment:
we should aim to throw descriptive errors throughout this function to handle them on the app-side like when we get the `feeRate` or when we select the outputs for the tx

Reply:
Could we rather get rid of the try catch block? The function should be throwing an error and not a string, as it is currently doing. If we make that change, then the try/catch block is redundant here.